internal/testhelpers/e2e/session.go

```diff
@@ -193,6 +193,7 @@ func (s *Session) SpawnCmdWithOpts(exe string, opts ...SpawnOptions) *termtest.C
 			RetainWorkDir:  true,
 			ObserveExpect:  observeExpectFn(s),
 			ObserveSend:    observeSendFn(s),
+			// ExtraOpts:      []expect.ConsoleOpt{expect.WithStdout(os.Stdout)},
```

Comment:
Let's leave a comment here to explain why this code is commented out and why you may want to enable it